pegasus_app/views.py

```diff
@@ -1,22 +1,61 @@
+from django import forms
+from django.http import HttpResponseRedirect
 from django.shortcuts import render
+from django.views.generic import UpdateView
 
 from pegasus_app.forms import PhoneNumberCheckConfigForm, SchedulesDayFormset
-from pegasus_app.models import ScheduleDay
+from pegasus_app.models import ScheduleDay, PhoneNumberCheckConfig
 
 
-def home_page(request):
+def change_config_number(request, id):
+    phone_config = PhoneNumberCheckConfig.objects.get(id=id)
+    if request.method == 'POST':
+        phone_config_form = PhoneNumberCheckConfigForm(request.POST or None, instance=phone_config)
+        schedule_formset = SchedulesDayFormset(request.POST or None, instance=phone_config)
+        if phone_config_form.is_valid():
+            phone_config = phone_config_form.save(commit=False)
+            phone_config.user_plan = request.user.plan
```

Comment:
No need to send plan twice, we have not two places to save a single entity: in user model and in phone config.

Also user plan is not conceptually located in phone config. Phone number does not have a plan. We should not save plan in phone number config. The user has a plan. We should save a plan in user config only.